skins/moono-lexicon/icons/icons.json

```diff
@@ -4,36 +4,59 @@
         "align-image-center": "align-image-center",
         "align-image-left": "align-image-left",
         "align-image-right": "align-image-right",
-        "anchor": "flag-full",
+        "anchor": {
+            "ltr": "flag-full"
+        },
         "blockquote": "quote-right",
         "bold": "bold",
-        "bulletedlist": "list-ul",
+        "bulletedlist": {
+            "ltr": "list-ul"
+        },
         "copy": "paste",
         "cut": "cut",
-        "find": "search",
+        "find": {
+            "ltr": "search",
+            "rtl": "search"
+        },
         "hidden": "hidden",
         "horizontalrule": "separator",
         "image": "picture",
-        "indent": "indent-more",
+        "indent": {
+            "ltr": "indent-more",
+            "rtl": "indent-less"
+        },
         "italic": "italic",
         "justifyblock": "align-justify",
         "justifycenter": "align-center",
         "justifyleft": "align-left",
         "justifyright": "align-right",
         "link": "link",
         "maximize": "full-size",
-        "numberedlist": "list-ol",
-        "outdent": "indent-less",
-        "redo": "redo",
+        "numberedlist": {
+            "ltr": "list-ol"
+        },
+        "outdent": {
+            "ltr": "indent-less",
+            "rtr": "indent-more"
+        },
+        "redo": {
+            "ltr": "redo",
```

Comment:
Are you sure these are directional? It kinda makes sense... but I'd love to get some reading about it :D

Reply:
CKEditor have it as directional (automatically uses `.cke-ltr` or `.cke-rtl` classes`). I assumed it was correct.